Branch-SDK/BranchOpenRequest.h

```diff
@@ -12,6 +12,7 @@
 @interface BranchOpenRequest : BNCServerRequest
 
 @property (nonatomic, copy) callbackWithStatus callback;
+@property (assign, nonatomic) BOOL clearLocalURL;
```

Comment:
Kindof a minor thing, usually we use YES and NO instead of TRUE and FALSE. I don't think it makes a difference here.

Reply:
This comment was for when we use clearLocalURL later on.

Reply:
In my testing this doesn't quite work as expected. The main issue is that repeated paste button presses result in copying the url again. We could try clearing it from the pasteboard when we're certain it's a branch link. Another option is to check if the local_url is the same, then ignore.